docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -452,67 +438,46 @@ Emotion's **css()** method works seamlessly with Material-UI.
 ```jsx
 /** @jsx jsx */
 import { jsx, css } from '@emotion/core';
-import Button from '@material-ui/core/Button';
+import Slider from '@material-ui/lab/SliderStyled';
 
 export default function EmotionCSS() {
   return (
     <div>
-      <Button>Default</Button>
-      <Button
+      <Slider defaultValue={30} />
+      <Slider
+        defaultValue={30}
         css={css`
-          background-color: #6772e5;
-          color: #fff;
-          box-shadow: 0 4px 6px rgba(50, 50, 93, 0.11), 0 1px 3px rgba(0, 0, 0, 0.08);
-          padding: 7px 14px;
-          &:hover {
-            background-color: #5469d4;
+          color: #20b2aa;
+          :hover {
+            color: #2e8b57;
           }
         `}
-      >
-        Customized
-      </Button>
+      />
     </div>
   );
 }
 ```
 
-### Controlling priority ⚠️
-
-**Note:** JSS injects its styles at the bottom of the `<head>`. If you don't want to mark style attributes with **!important**, you need to change [the CSS injection order](/styles/advanced/#css-injection-order), as in the demo:
-
-```jsx
-import { StylesProvider } from '@material-ui/core/styles';
-
-<StylesProvider injectFirst>{/* Your component tree.
-      Now, you can override Material-UI's styles. */}</StylesProvider>;
-```
-
 ### Theme
 
 Material-UI has a rich theme structure that you can leverage for
 the color manipulations, the transitions, the media queries, and more.
 
+By using the Material-UI's Provider, the theme will be available for the
+context as well (emotion or styled-components, depending on your configuraiton).
+
 We encourage to share the same theme object between Material-UI and your styles.
 
 ```jsx
-<Button
+<Slider
+  defaultValue={30}
   css={(theme) => css`
-    background-color: ${theme.palette.primary.main};
-    color: #fff;
-    box-shadow: 0 4px 6px rgba(50, 50, 93, 0.11), 0 1px 3px rgba(0, 0, 0, 0.08);
-    padding: 4px 10px;
-    font-size: 13px;
-    &:hover {
-      background-color: ${darken(theme.palette.primary.main, 0.2)};
-    }
-    ${theme.breakpoints.up('sm')} {
-      font-size: 14px;
-      padding: 7px 14px;
+    color: ${theme.palette.primary.main};
+    :hover {
+      color: ${darken(theme.palette.primary.main, 0.2)};
     }
   `}
->
-  Customized
-</Button>
+/>
 ```
```

Comment:
Does emotion need a different section to https://deploy-preview-23030--material-ui.netlify.app/guides/interoperability/#theme? Or could we link styled-components as we do for the styled() API?

Reply:
Not really, let's link the same for simplicity